interface/usergroup/mfa_totp.php

```diff
@@ -0,0 +1,218 @@
+<?php
+error_reporting(E_ALL);
+ini_set('display_errors', 1);
```

Comment:
Remove above 2 lines

Reply:
you can use user debug in Administration->Globals->Logging